__tests__/generate-from-template.spec.js

```diff
@@ -146,4 +150,15 @@ describe('generateFromTemplate', () => {
     expect(console.log).toHaveBeenCalledTimes(1);
     expect(console.log).toHaveBeenNthCalledWith(1, 'postGenerationMessageMock');
   });
+
+  it('should not store responses if generatorOptions.doNotStoreResponses is true', async () => {
```

Comment:
The inverse should also be tested

Reply:
resolved in 26a819b549befc016bc6480f3e074b9597bc3a12